travis-fix-coverage.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
+
+cat > .coveragerc <<EOF
+[paths]
+source =
+    .
+    $PWD
+EOF
+mv .coverage .coverage.merge
+coverage combine --append .
```

Comment:
Is this related to the command one line above? Can this and previous line be written as
```bash
if [[ -f .coverage ]]; then
  mv .coverage .coverage.merge
  coverage combine --append .
fi
```
?